util.js

```diff
@@ -1057,6 +1057,16 @@ export const getTransferables = o => {
   _recurse(o);
   return result;
 };
+export function getDiffQuaternion(quaternionA, quaternionB) {
```

Comment:
For this function it's better to pass in the `target` quaternion from the caller rather than return the util local.

It makes it harder to make variable aliasing mistakes, which can be very hard to debug later.

Reply:
Changed https://github.com/webaverse/app/pull/2731/commits/fd2aa56664b0fd41cc6f74bc32f568b57d045fe7